test/parallel/test-debug-port-zero-cluster.js

```diff
@@ -0,0 +1,28 @@
+// Flags: --debug=0
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+const cluster = require('cluster');
+
+if (cluster.isMaster) {
+  const ports = [];
+  for (const worker of [cluster.fork(), cluster.fork()]) {
```

Comment:
Maybe I'm wrong here, but with current cluster algorithm we will start workers to listen on ports `[1, 2]`?